Makefile

```diff
@@ -0,0 +1,67 @@
+.DEFAULT_GOAL := test
+
+.PHONY: clean compile_translations dummy_translations extract_translations fake_translations help html_coverage \
+	migrate pull_translations push_translations quality requirements test update_translations validate
+
+help:
+	@echo "Please use \`make <target>' where <target> is one of"
+	@echo "  clean                      delete generated byte code and coverage reports"
+	@echo "  compile_translations       compile translation files, outputting .po files for each supported language"
+	@echo "  dummy_translations         generate dummy translation (.po) files"
+	@echo "  extract_translations       extract strings to be translated, outputting .mo files"
+	@echo "  fake_translations          generate and compile dummy translation files"
+	@echo "  help                       display this help message"
+	@echo "  html_coverage              generate and view HTML coverage report"
+	@echo "  migrate                    apply database migrations"
+	@echo "  pull_translations          pull translations from Transifex"
+	@echo "  push_translations          push source translation files (.po) from Transifex"
+	@echo "  quality                    run PEP8 and Pylint"
+	@echo "  requirements               install requirements for local development"
+	@echo "  test                       run tests and generate coverage report"
+	@echo "  validate                   run tests and quality checks"
+	@echo ""
+
+clean:
+	find . -name '*.pyc' -delete
+	coverage erase
+	rm -rf assets
+
+local-requirements:
+	pip install -qr requirements/local.txt --exists-action w
+
+requirements:
```

Comment:
I'm not convinced that `make requirements` should be local and `make prod-requirements` should be production rather than `make local-requirements` being local. Although, looking at the help text above, you should either fix the docs or fix the code.